### DISCUSSION TITLE
Welcome to rempsyc Discussions!

### DISCUSSION THREAD

**Discussion by rempsyc** [rempsyc/rempsyc#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hugo-v587** [comment]
Hi!
My name is Hugo. I am a social science researcher and I am using your rempsyc package to generate publication-ready t-tests. Your package is amazing! However, I've got the following error message when running the syntax from your [intro document](https://rempsyc.remi-theriault.com/articles/t-test):

```r
nice_t_test(data = mtcars,
            response = names(mtcars)[1:6],
            group = "am",
            warning = FALSE) -> t.test.results

t.test.results

my_table <- nice_table(t.test.results)

my_table

```
And the error message is:

```r
`Error in get_columns_id(x[[part]], j) : 
  invalid columns selection: unknown selection type`

```
I am using R version 3.6.3 (2020-02-29) and that may need to be updated.

**Reply by rempsyc** [reply]
Hi @Hugo-v587! Thanks for using my package!

That example works for me. However, `nice_table` relies on the `flextable` package. What is your version of `flextable`? Can you try updating `flextable` to the latest CRAN version? Thanks!

**Comment by rempsyc** [comment]
Hi Hugo! Thanks for using my package!

That example works for me. However, `nice_table` relies on the `flextable` package. What is your version of `flextable`? Can you try updating `flextable` to the latest CRAN version? Thanks!

**Reply by Hugo-v587** [reply]
Hi rempsyc,

I've updated the R version and the flextable package and now it works. Thanks!

Your package is quite useful!
